.github/ISSUE_TEMPLATE/missing-layer-type.md

```diff
@@ -7,8 +7,10 @@ assignees: ''
 
 ---
 
+# Missing Layer Type
```

Comment:
I don't think we need this extra text.

When creating an issue with this template, "Missing Layer Type" is already displayed above the web form. If we add it to this text, then it will be part of the issue that gets created, unless the user deletes the text before submitting.

I think "Please fill out these questions" is implied. It's also more text that will become part of the issue unless the user deletes it.